### PR TITLE
Unescaped passwords to allow special characters

### PR DESCRIPTION
**Pull request by yovasx2**
@tubbo: The implementation of specs for #280

### CODE REVIEW

File: test/redis/store/factory_test.rb [hunk 1]
```diff
@@ -1,103 +1,128 @@
...
+
+      it 'uses specified namespace' do
+        store = Redis::Store::Factory.create 'redis://127.0.0.1:6379/0/theplaylist'
+        store.to_s.must_equal('Redis Client connected to 127.0.0.1:6379 against DB 0 with namespace theplaylist')
       end
 
-      it "uses specified port" do
-        store = Redis::Store::Factory.create "redis://127.0.0.1:6380"
-        store.to_s.must_equal("Redis Client connected to 127.0.0.1:6380 against DB 0")
+      it 'uses specified password' do
```
Comment (yovasx2) [inline test/redis/store/factory_test.rb]: These are the new specs

**Comment by tubbo** [comment]
@yovasx2 please

**Comment by yovasx2** [comment]
But I cannot edit the other PR cause is not mine, so u want me to extract just the password change?

**Comment by tubbo** [comment]
@yovasx2 that was my mistake, I thought you were the same person as the author of #280...anyway, yeah I'd like it if the password escaping change was isolated from the other style improvements.

**Comment by yovasx2** [comment]
no problem, I just drop my styles changes, I think you like more your style jajajaja.
Here new PR isolated #286

**Comment by tubbo** [comment]
hah, honestly the style to me isn't important...it's just harder to review code when there's a lot of noise in the diff.

**Pull request closed without merging**